### PR TITLE
Fixes #11: Add support for Terraform v0.12

### LINKED ISSUE #11: Add support for Terraform v0.12
I'm getting this error,
```
Error: Unsupported block type

  on .terraform/modules/datastore/terraform-google-modules-terraform-google-cloud-datastore-967e958/main.tf line 28, in resource "null_resource" "cloud-datastore-indices":
  28:   triggers {

Blocks of type "triggers" are not expected here. Did you mean to define
argument "triggers"? If so, use the equals sign to assign it a value.
```
terraform --version
```
Terraform v0.12.2
+ provider.google v2.9.1
+ provider.local v1.2.2
+ provider.null v2.1.2
```

### PR DESCRIPTION
**Pull request by nick4fake**
https://github.com/terraform-google-modules/terraform-google-cloud-datastore/issues/11

### CODE REVIEW

**Comment by nick4fake** [comment]
datastore-tests
datastore-lint

File: Makefile [hunk 1]
```diff
@@ -15,19 +15,17 @@
...
+DOCKER_ORG 				:= gcr.io/cloud-foundation-cicd
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM 	?= 2.3.0
+DOCKER_REPO_BASE_KITCHEN_TERRAFORM 	:= ${DOCKER_ORG}/cft/kitchen-terraform:${DOCKER_TAG_BASE_KITCHEN_TERRAFORM}
 
 # All is the first target in the file so it will get picked up when you just run 'make' on its own
-all: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace generate_docs docker_build_terraform docker_build_kitchen_terraform
+all: check generate_docs
+
+.PHONY: check
+check: check_shell check_python check_golang check_terraform check_docker check_base_files test_check_headers check_headers check_trailing_whitespace
```
Comment (kopachevsky) [inline Makefile]: why are we skipping check_generate_docs?

File: test/make.sh [hunk 2]
```diff
@@ -36,6 +36,32 @@ function basefiles() {
...
+  # stderr is silently redirected to avoid console log spam.
+  if xargs --no-run-if-empty </dev/null 2>/dev/null; then
+    compat=("--no-run-if-empty")
+  fi
+  xargs "${compat[@]}" "$@"
+}
+
 # This function runs the hadolint linter on
 # every file named 'Dockerfile'
 function docker() {
```
Comment (ivankorn) [inline test/make.sh]: this function is deprecated and to be removed

File: Makefile [hunk 3]
```diff
@@ -56,10 +54,6 @@ check_docker:
 check_base_files:
```
Comment (ivankorn) [inline Makefile]: the docker linter [target](https://github.com/terraform-google-modules/terraform-google-cloud-datastore/pull/12/files#diff-b67911656ef5d18c4ae36cb6741b7965R49-R51) is deprecated and to be removed

File: Makefile (hunk 1 above)
Comment (nick4fake) [reply Makefile]: There was no such linter

**Pull request merged**